### PR TITLE
Add IL sfsites fetcher

### LINKED ISSUE #149: fetch sfsites from Illinois
[learn our pipeline: fetch](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Runner-pipeline-stages#fetch)

Fetch data from: https://coronavirus.illinois.gov/s/statewide-vaccination-locations and https://coronavirus.illinois.gov/s/vaccination-location

Put your script in a file named: il/sfsites/fetch.sh

Store it (without processing) in a new file created in the directory passed as the first argument (`sys.argv[1]`).

Check the wiki to learn more about the purpose of the fetch stage and how to get set up for development!

### Tips

1. While working on your code, run it at any point:
    ```sh
    poetry run vaccine-feed-ingest fetch <state>/<site>
    ```

### PR DESCRIPTION
**Pull request by yotann**
fixes #149

### CODE REVIEW

File: vaccine_feed_ingest/runners/il/sfsites/fetch.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        },
+    )
+
+    response.raise_for_status()
+    with open(os.path.join(output_dir, output_filename), "w") as f:
+        f.write(response.text)
+
+
+call_api("statewide.json", "getMassVaccSites")
+call_api("locations.json", "getSites", {"countOfMiles": "10", "selectedCity": None})
```
Comment (eliblock) [inline vaccine_feed_ingest/runners/il/sfsites/fetch.py:63]: Do we need to call this multiple times with various inputs to get good coverage of the state, or is `{"countOfMiles": "10", "selectedCity": None}` a secret way to say "show all the sites"?

**Review by eliblock — APPROVED** [review]
Thank you for adding this - I'm excited to get this data!

I asked one question about if we need to send multiple queries, but other than that, this looks good to go!

File: vaccine_feed_ingest/runners/il/sfsites/fetch.py (hunk 1 above)
Comment (yotann) [reply vaccine_feed_ingest/runners/il/sfsites/fetch.py:63]: It provides all sites (since there's no city selected).

Comment (tmb) [reply vaccine_feed_ingest/runners/il/sfsites/fetch.py:63]: Tested — it pulls from across the state in this form.

**Comment by tmb** [comment]
Looks good! Merging.

**Pull request merged**